sys/universal_address/universal_address.c

```diff
@@ -248,31 +254,31 @@ int universal_address_compare_prefix(universal_address_container_t *entry,
 
     /* Get the index of the first trailing `0` */
     int i = 0;
-    for( i = entry->address_size-1; i >= 0; --i) {
-        if( prefix[i] != 0 ) {
+    for (i = entry->address_size-1; i >= 0; --i) {
+        if (prefix[i] != 0) {
             break;
         }
     }
 
-    if( memcmp(entry->address, prefix, i) == 0 ) {
+    if (memcmp(entry->address, prefix, i) == 0) {
         /* if the bytes-1 equals we check the bits of the lowest byte */
         uint8_t bitmask = 0x00;
         /* get a bitmask for the trailing 0b */
-        for( uint8_t j = 0; j < 8; ++j ) {
-            if ( (prefix[i] >> j) & 0x01 ) {
+        for (uint8_t j = 0; j < 8; ++j) {
+            if ((prefix[i] >> j) & 0x01) {
                 bitmask = 0xff << j;
                 break;
             }
         }
 
-        if( (entry->address[i] & bitmask) == (prefix[i] & bitmask) ) {
+        if ((entry->address[i] & bitmask) == (prefix[i] & bitmask)) {
             ret = entry->address[i] != prefix[i];
-            if( ret == 0 ) {
+            if (ret == UNIVERSAL_ADDRESS_EQUAL) {
                 /* check if the remaining bits from entry are significant */
                 i++;
-                for(; i < entry->address_size; ++i) {
-                    if( entry->address[i] != 0 ) {
-                        ret = 1;
+                for ( ; i < entry->address_size; ++i) {
```

Comment:
I intentionally added the whitespace before the first `;` in the `for` statement because I tend to overlook them in such cases.